modules/phase_field/include/postprocessors/FeatureFloodCount.h

```diff
@@ -65,14 +66,14 @@ class FeatureFloodCount :
   };
 
   // Retrieve field information
-  virtual Real getEntityValue(dof_id_type entity_id, FIELD_TYPE field_type, unsigned int var_idx=0) const;
+  virtual Real getEntityValue(dof_id_type entity_id, FieldType field_type, unsigned int var_idx=0) const;
 
   virtual const std::vector<std::pair<unsigned int, unsigned int> > & getElementalValues(dof_id_type elem_id) const;
 
   inline bool isElemental() const { return _is_elemental; }
 
   /// This enumeration is used to indicate status of the grains in the _unique_grains data structure
-  enum STATUS
+  enum class Status
```

Comment:
I hope all our minimum required compilers support `enum class`, this is another thing we never tested.